generate_sharktank.py

```diff
@@ -239,11 +265,11 @@ def is_valid_file(arg):
     if args.torch_model_csv:
         save_torch_model(args.torch_model_csv)
 
-    if args.tf_model_csv:
-        save_tf_model(args.tf_model_csv)
+    # if args.tf_model_csv:
```

Comment:
Is this commenting intentional? I thought these args were enabled by default and the script needed them.

Reply:
sorry this is for testing only, good catch